libs/@guardian/ab-react/tsconfig.json

```
@@ -1,5 +1,7 @@
 {
 	"extends": "../../../tsconfig.base.json",
-	"files": [],
-	"include": ["."]
+	"include": ["."],
+	"compilerOptions": {
+		"tsBuildInfoFile": "tsconfig.tsbuildinfo"
+	}
```

```suggestion
	"include": ["."]
```
lets remove incremental from our tsconfig